one-ui/src/main/java/com/marklogic/hub/oneui/controllers/EnvironmentController.java

```diff
@@ -122,4 +126,16 @@ public JsonNode reset() {
         obj.put("reset", true);
         return obj;
     }
+
+    @RequestMapping(value = "/project-info", method = RequestMethod.GET)
+    @ResponseBody
+    public JsonNode getProjectInfo(HttpSession session) {
```

Comment:
What about tests for this? We'd at least want a smoke test to verify that there's not an e.g. obvious null-pointer exception. 

Also, I think @hao1st  mentioned this too - based on the current design of oneui, couldn't HubProject be part of EnvironmentService? That would start removing one concern from HubConfig, which is overloaded with concerns right now. I think this would be a good move, as the HubProject is a global object that is part of the environment - it's not specific to a user.

Reply:
1. I will add a test .
2. I don't think we can autowire HubProject in EnvironmentService. The first object created in one-ui is HubConfigSession and that is what we are using through out and that's why I had to access HubProject from HubConfigSession

Reply:
If HubProjectImpl is autowired in, then it's always available, right?

I'll approve the PR, I just feel strongly that we need to stop using HubConfig, which is doing far too much, and use HubProject and a doesnt-yet-exist HubClient when possible.